redash/query_runner/big_query.py

```diff
@@ -132,6 +136,7 @@ def _get_total_bytes_processed(self, jobs, query):
         job_data = {
             "query": query,
             "dryRun": True,
+            "useLegacySql": self.configuration.get('useLegacySql', False),
```

Comment:
And because it goes unchecked by default, let's change the configuration name to "Use Standard SQL (Beta)".